### DISCUSSION TITLE
Getting gram decompostion

### DISCUSSION THREAD

**Discussion by JOlucak** [iFR-ACSO/casos#86] [Q&A]
Assume we solved a SOS problem. We are now interested to get the SOS decomposition of a constrait i.e. z^T Q z 

Is it currently possible to get this from the solver output? Or does CaSoS already provide a function that might compute such a decomposition?

**Comment by tcunis** [comment]
At the moment, the only way to obtain the Gram decomposition is by doing the SOS-to-SDP relaxation by hand. It would indeed be possible to provide the Gram decomposition, if existing, upon solution. The best place to do so would be the `stats` field of the SOS solver.
